thesis-de/content-de.tex

```diff
@@ -1094,56 +1094,221 @@ \subsubsection{Durchführung Evaluierung Frontend}
+an bestimmte Kundenumgebungen. Zusätzlich zu dem Quellcode, können hier
+zusätzliche Plugins in der Porgrammiersprache Java entwickelt und in das System
+eingepflegt werden, wenn hier weitere Graphen, Analysen oder ähnlichem benötigt
+werden. Grafana hat eine stetig wachsende Community mit sämtlichen Dokumenten
+zur Einrichtung oder Erweiterung des Systems. Der gesamte Quellcode ist auf
+GitHub veröffentlicht, somit haben auch andere Mitglieder von GitHub, die 
+Möglichkeit, das Produkt zu verbessern und neue Funktionen zu implementieren.
+Dies bedarf lediglich die Zustimmung, durch die Mitglieder des Produktes auf
+GitHub. Die vom Anwender erstellen Dashboards, mit allen Graphen und Statements,
+können vom System, wie auch vom Anwender selber auf einen Externen
+Dateifreigabe Server oder Lokal abgespeichert werden.
+
+Zusammenfassung:
 \begin{outline}
-  \1 Grafana
-  + Authentisierung, Authentifizierung und Autorisierung
-  + Gute Community und eine vielzahl an Dokumenten
-  + Leichte Anpassungen an Graphen und Einstellungen
-  + Dashboards können auf externen Server gesichert werden
-  + Grafana kann mittels bereitsgestellter API arbeiten
-  + Bietet eine Plugin Erweiterung an
-
-  + Multiplattform Support
-  + OpenSource
-
-  \textasciitilde Webdesign kann nicht angepasst werden
-  \textasciitilde Webtemplate nur aus zwei auswählbar (dark/white)
-
-  -
-  -
-  -
-
-  \1 Graphite
-  + Authentisierung, Authentifizierung und Autorisierung
-  + Gute Community und eine vielzahl an Dokumenten
-  +
-
-
-  \1 Zabbix-frontend
-
-
-
-  \1 Datadog
-
-
-
-  \1 Gridster-D3
-  +
-  +
-  +
-
-  \textasciitilde
-  \textasciitilde
-
-  - Lizenz muss Monatlich erworben werden
-  -
+  Positiv:
+  \1 Authentisierung, Authentifizierung und Autorisierung
+  \1 Gute Community und eine vielzahl an Dokumenten durch GitHub
+  \1 Leichte Anpassungen an Graphen und Einstellungen möglich
+  \1 Dashboards können auf externen Server oder Lokal gesichert werden
+  \1 API Unterstüzung
+  \1 Plugin Erweiterung möglich (Java)
+  \1 OpenSource Softwarelösung
+
+  Negativ:
+  \1 Plugin Erweiterung nur in Programmiersprache Java möglich
+  \1 Webdesign kann nicht angepasst werden
+  \1 Nur zwei Webtemplates verfügbar (dark/white)
+\end{outline}
 
+\paragraph{Graphite}
+\label{paragraph:graphite}
+Die Softwarelösung Graphite, wurde von Chris Davis im Jahr 2006 Entwickelt und
+gehört zu einer der meist eigesetzten Softwarelösungen im Bereich Monitoring
+und Visualisierung. Graphite ist wie auch Grafana eine Open Source Lösung und
+ermöglicht es dem Anwender die Software bearbeiten und anpassen zu können. Es 
+verfügt über eine inhaltlich gute und umfangreiche Community, mit einer eigenem
+Wiki Board, wo alle relevatnen Dokumentationen enthalten sind. Graphen können
+leicht mittelts Statements über eine API oder direkt an das
+Datenhaltungssystem durchgeführt werden. Eine Anmeldung an der WebOberfläche,
+mittels Benutzername und Kennwort oder LDAP Anbindung ist ebenfalls möglich.
```

Comment:
Kein Plan was wieder LDAP ist, nimmst du als Fachwort in der Liste auf und verweist dann drauf?